doc/guide.md

```diff
@@ -2816,6 +2816,9 @@ leave it unmodified (default).
   Defaults to 10000.
 * `token_cache_ttl_ms`: Time to live in milliseconds for token cache entries.
   Defaults to 180000 (3 minutes).
+* `deploy.waitIterations`: Maximum number of iterations for deployment - before
```

Comment:
Environment variables with periods/dots in their names should be avoided because a period is not POSIX compliant and therefore some shells, notably, the BusyBox /bin/sh included in Alpine Linux, strip them.
References:
* warning in OpenJDK docs: https://hub.docker.com/_/openjdk/
* POSIX.1-2017: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html

Reply:
Not an environment variable. This section in the guide lists Java properties.